lib/vagrant-openshift/action.rb

```diff
@@ -153,6 +153,18 @@ def self.repo_sync_sti(options)
         end
       end
 
+      def self.repo_sync_jenkins(options)
```

Comment:
i know you're following existing patterns but it strikes me as weird we don't have a generic "sync repo" command where you just provide the local directory name (and assume it matches the remote name).  @danmcp @stevekuznetsov am i missing something?